server-common/src/main/java/com/datastrato/gravitino/server/auth/AuthenticationFilter.java

```diff
@@ -52,7 +53,8 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         authData = headerData.nextElement().getBytes(StandardCharsets.UTF_8);
       }
       if (authenticator.isDataFromToken()) {
-        authenticator.authenticateToken(authData);
+        Principal principal = authenticator.authenticateToken(authData);
+        request.setAttribute(AuthConstants.AuthenticatedPrincipalAttributeName, principal);
```

Comment:
Is it a common way to get user name from attribute?

Reply:
Pulsar uses this way. I don't find other implements. Some systems will put the  user formation into the session. But we don't have session manager in our Jetty server. It may bring some cost to add a session manager in our Jetty server. So I choose to use to put the user to the attributes.

Reply:
My original purpose is to use this:

<img width="419" alt="image" src="https://github.com/datastrato/gravitino/assets/850797/989ca8cf-f6b3-4b6a-be34-6aa641ccff61">

If you use another way, maybe you can remove that variable.

Reply:
> My original purpose is to use this:
> 
> <img alt="image" width="419" src="https://private-user-images.githubusercontent.com/850797/293030719-989ca8cf-f6b3-4b6a-be34-6aa641ccff61.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM2ODg0MTAsIm5iZiI6MTcwMzY4ODExMCwicGF0aCI6Ii84NTA3OTcvMjkzMDMwNzE5LTk4OWNhOGNmLWY2YjMtNGI2YS1iZTM0LTZhYTY0MWNjZmY2MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIyN1QxNDQxNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iYzI3ZjBiYzc1ZmZjYzA3MGE3MWFkMGRkMjU2MjUxMmQ3M2U1ZDA3Y2JlNTJiMGQ4YzQ0MWZkMzc2MzllYTU1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.RaLK-24FEI_T40O4crWPST9KwtBMs4b2fGS0zm61y_Q">
> If you use another way, maybe you can remove that variable.

I have used this variable. I need to set the attribute and get the attribute with this variable.